pkgs/applications/video/epgstation/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, makeWrapper, bash, nodejs }:
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+  workaround-opencollective-buildfailures = stdenv.mkDerivation {
+    # FIXME: This should be removed when a complete fix is available
+    # https://github.com/svanderburg/node2nix/issues/145
+    name = "workaround-opencollective-buildfailures";
+    phases = [ "installPhase" ];
+    installPhase = ''
+      mkdir -p $out/bin
+      touch $out/bin/opencollective-postinstall
+      chmod +x $out/bin/opencollective-postinstall
+    '';
+  };
+in
+nodePackages."epgstation-git+https://github.com/l3tnun/EPGStation.git#v1.7.4".override (drv: {
+  buildInputs = [ bash ];
+  nativeBuildInputs = [
+    workaround-opencollective-buildfailures
+    makeWrapper
+    nodePackages.node-pre-gyp
+  ];
+
+  preRebuild = ''
+    patch -p1 <<<"${builtins.readFile ./epgstation.patch}"
```

Comment:
Is there no regular `patchPhase` for node packages? Also, a comment should explain why this patch is necessary.

Reply:
`preRebuild` is the closest thing to `patchPhase` for node2nix packages. I've renamed the patch filename to a more descriptive one. I'll push the changes once I'm done fixing other parts of this PR.

Reply:
Change pushed